tslint.json

```diff
@@ -72,7 +72,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
We can remove this lint rule but I think we should update the tsconfig.json with:

    "compilerOptions": {
      "noUnusedLocals": true,                /* Report errors on unused locals. */
      "noUnusedParameters": true             /* Report errors on unused parameters. */
    }

..otherwise, we will lose its purpose.